charts/service/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: service
 description: A generic k8s service chart
 type: application
-version: 1.1.4
+version: 1.2.0
```

Comment:
i'm not aware of all consequences of this change, and therefor version update that this PR should introduce. are any services relying on port that was defined in `charts/service/values.yaml`?  those serve as default values, right?

Reply:
other apps will refer to this chart like so:
https://github.com/codecademy-engineering/Codecademy/blob/develop/.helm/codecademy/requirements.yaml#L7

Reply:
got it! but just to confirm, removing [this line](https://github.com/codecademy-engineering/helm-charts/blob/main/charts/service/values.yaml#L51) won't break anything?

Reply:
for context, we talked and v1.2.0 makes more sense since we're removing default value